### PR TITLE
Add enhanced relative time to variable tab cells

### PR DESCRIPTION
**Pull request by Destroy666x**
Adds more optimal time units display to variable tab.

Could be even better like "5 minutes 20 seconds ago" and considering singular/plural, but maybe someday, didn't bother trying due to potential space contraints, this is a lot better and more readable than it was with decimal anyways. Also, before you move the function out of new file - even if you think this isn't global enough as of now, I plan to reuse it in another upcoming PR with a new functionality related to macros, so please don't :P

### CODE REVIEW

**Comment by Destroy666x** [comment]
https://github.com/WarmUpTill/SceneSwitcher/issues/1043

File: lib/utils/time-helpers.hpp [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#pragma once
+#include <QString>
+
+namespace advss {
+
+QString FormatRelativeTime(const float seconds);
```
Comment (WarmUpTill) [inline lib/utils/time-helpers.hpp:6]: Sorry if I  just overlooked it, but why are you using a `float` here and not `double` / `uint64_t`?

**Comment by WarmUpTill** [comment]
It is probably related to the other PR you mentioned but do you think conversion to weeks, months, and years are really required?

Also maybe it might be worth changing the magic numbers for month and year to:

>Years is equal to 365.2425 days (the average length of a Gregorian year). months is equal to 30.436875 days (exactly 1/12 of years)

And just to mention it as an alternative approach:
The function `Variable::GetSecondsSinceLastUse()` could be reworked to return `std::chrono::duration` to make conversion to other units easier.

Those are not mandatory changes for me to merge the PR.
Just some thoughts / suggestions.
Sorry, if those suggestions conflict with your other upcoming PR.

**Comment by Destroy666x** [comment]
Not with the new PR, but weeks could be useful even in the tab itself, if something is running long-term - no idea if people are using OBS for sort of 24/7 online TV-like stuff, together with variables from this plugin. The function could also later be turned into OBS function for relative time, together with some other date/time conversions maybe. Not in my plans, but yeah.

> Also maybe it might be worth changing the magic numbers for month and year to:

Ah, good to know those are used measurements, I'll change to that.

> The function Variable::GetSecondsSinceLastUse() could be reworked to return std::chrono::duration to make conversion to other units easier.

Not necessarily for me. But you're right it's likely a better fit, feel free to edit this if you'd like.

**Pull request merged**